bloom-housing/bloom
# feat: min max occupancy validation in Add Unit

# Pull Request Template

## Issue Overview

This PR addresses #2622 


## Description

In New Listing --> Add Unit, you were previously able to successfully add a Min Occupancy larger than the Max Occupancy. Now both Min and Max Occupancy have errors when min is greater than max upon attempting to Save & New or Save & Exit.

![image](https://user-images.githubusercontent.com/60076088/163234819-081fc0b3-fb64-45d5-9ed4-034e4b3c7032.png)

## How Can This Be Tested/Reviewed?

Entry Point: http://localhost:3001/listings/add

1. Go to New Listing in partners
2. Click the Add Unit link to open the Add Unit Modal
3. In the Min Occupancy dropdown, select a number
4. In the Max Occupancy dropdown, select a number less than the number you chose in Min Occupancy.
5. Press Save & Exit - you will see errors under both fields

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
